Tests/XCTest/XCTestExtensions.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  XCTestExtensions.swift
+//  SwifterSwift
+//
+//  Created by Guy Kogus on 28/7/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(XCTest)
+import XCTest
+
+#if canImport(UIKit)
+import UIKit
+/// SwifterSwift: Color
+public typealias Color = UIColor
+#endif
+
+#if canImport(AppKit) && !targetEnvironment(macCatalyst)
+import AppKit
+/// SwifterSwift: Color
+public typealias Color = NSColor
+#endif
+
+#if canImport(AppKit) || canImport(UIKit)
+/// SwifterSwift: Asserts that two `Color`s are equal within a certain accuracy.
```

Comment:
I think we should document better what exactly the accuracy means, like what is the measurement used for determining a color value, in this case, the accuracy is applied on RGB values... because a Float accuracy in color does not make clear the behavior of the function, so it is important that is documented.